install/kubernetes/helm/istio/files/injection-template.yaml

```
@@ -44,19 +50,20 @@ initContainers:
 {{- end }}
   securityContext:
     allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
+    privileged: {{ .Values.global.proxy.privileged }}
     capabilities:
+  {{- if not .Values.istio_cni.enabled }}
       add:
       - NET_ADMIN
       - NET_RAW
```

`iptables` needs to have the capability to load modules dynamically. Can you try to add `SYS_MODULE` explicitly?
```suggestion
       add:
       - NET_ADMIN
       - NET_RAW
       - SYS_MODULE
```

---

Thanks! Updating

---

TIL